contracts/modules/staking/StakingPool.sol

```diff
@@ -55,7 +55,7 @@ contract StakingPool is IStakingPool, Multicall {
   uint96 public rewardPerSecond;
 
   uint40 poolId;
-  uint24 public nextAllocationId;
+  uint24 public nextAllocationId = 1;
```

Comment:
This will be stored in the storage of the implementation, not the proxy, so it doesn't work

Reply:
Resolved.